corehq/apps/app_manager/templates/app_manager/partials/releases/releases_table.html

```diff
@@ -75,7 +75,7 @@
     <p>
       {% blocktrans %}
         For more information on deprecated case types, see the following
-        <a target="_blank" href="https://confluence.dimagi.com/display/commcarepublic/Deprecating+Case+Types+and+Case+Properties+using+Data+Dictionary">documentation</a>.
+        <a target="_blank" href="https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143944977/Data+Dictionary">documentation</a>.
```

Comment:
Looks like there's a more specific section on deprecating? https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143944977/Data+Dictionary#Deprecating-%26-Delete-Case-Types-and-Case-Properties